ipn/serve.go

```diff
@@ -38,27 +37,11 @@ type ServeConfig struct {
 	// AllowFunnel is the set of SNI:port values for which funnel
 	// traffic is allowed, from trusted ingress peers.
 	AllowFunnel map[HostPort]bool `json:",omitempty"`
-}
 
-// StripEphemeral returns a copy of the ServeConfig that ensures
-// that ephemeral TCP connections stay in memory only by removing
-// both TCP and web handlers where the Ephemeral field is set to true.
-func (s *ServeConfig) StripEphemeral() *ServeConfig {
-	conf := s.Clone()
-	maps.DeleteFunc(conf.Web, func(hp HostPort, wsc *WebServerConfig) bool {
-		p, _ := hp.Port()
-		tcpHandler := conf.TCP[p]
-		return tcpHandler != nil && tcpHandler.Ephemeral
-	})
-	maps.DeleteFunc(conf.AllowFunnel, func(hp HostPort, b bool) bool {
-		p, _ := hp.Port()
-		tcpHandler := conf.TCP[p]
-		return tcpHandler != nil && tcpHandler.Ephemeral
-	})
-	maps.DeleteFunc(conf.TCP, func(u uint16, th *TCPPortHandler) bool {
-		return th.Ephemeral
-	})
-	return conf
+	// Foreground is a map of an IPN Bus session id to a
+	// foreground serve config. Note that only TCP and Web
+	// are used inside the Foreground map.
```

Comment:
we'll want to flesh out this comment more :)

Reply:
Done, added more words. But happy to change/add if needed